### DISCUSSION TITLE
digital clock and quick move and quick drop.

### DISCUSSION THREAD

**Discussion by Gaiie** [BedrockTweaks/Files#337] [Ideas]
![Picsart_24-09-14_17-46-34-293](https://github.com/user-attachments/assets/f735f6dd-a186-4336-afd0-304fe04fb4a6)
![Picsart_24-09-14_17-47-23-582](https://github.com/user-attachments/assets/8031063f-b97d-4a10-a420-48b2462b2eac)

**Comment by PAPISOP** [comment]
Visible compass should only be appeared in the hud when the player has the compass item in their inventory or else it will not appear (if it does appear without it being in the inventory then it will be considered a *cheat*)
